### DISCUSSION TITLE
ElectRs is indexing blockchain since days-How to control the Progress

### DISCUSSION THREAD

**Discussion by quarreller** [raspiblitz/raspiblitz#4856] [Q&A]
I'm not sure if ElectRs is running and indexing correctly.
It's never taken this long before.

Is there a way to show me in the menu or on the console whether ElectRs is running and indexing correctly or what percentage has been indexed so far?

Many thanks for any information

**Comment by rootzoll** [comment]
On terminal you can run the following command to get more status info on the sync:
`/home/admin/config.scripts/bonus.electrs.sh status-sync`

**Reply by quarreller** [reply]
Thank you for your answer.
After 2 more days I just rebooted and then ElectRs showed me the QR code for my mobile wallet.

The result of your mentioned command is now this:

serviceRunning=1
electrumResponding=1
blockheight=‘0’
blockheightPercent=‘0’
initialSynced=1

I think the 1 stands for ElectRs is running and synced.
But why are the two zeros there?

Thanks for your time

**Reply by rootzoll** [reply]
the script tries to get information on synced blockheight/percent .. but if thats failes it shows the valaue zero
